packages/block-library/src/gallery/style.scss

```diff
@@ -148,4 +148,12 @@
 	&.alignright {
 		display: flex;
 	}
+
+	// If the gallery is centered, center the content inside as well.
+	[data-align="center"] &,
```

Comment:
Hi @jasmussen is `[data-align="center"] &`really needed? It looks like even in the editor the align center class is being applied.
If it is really needed I think we should add this selector in editor.scss. I think this selector will never be used on the frontend and is editor specific, so we should avoid sending editor specific CSS to sites frontend.
I checked that we have other editor selectors in this file we should probably remove them as well maybe in other PR as it is not related to this changes.

Reply:
The data-align properties, I'm almost completely convinced, _used_ to be necessary in order for the styles to take on both front and editor, as in editor those alignment classes were not present. I may be misremembering, but I'm almost certain, which is why those classes were also present for left and right alignments. Those are all removed now, in https://github.com/WordPress/gutenberg/pull/11040/commits/fedf089d03e90ae2fc2510a944030bc34d02b354, because as you note, they seem to be present, and it's working:

![screenshot 2018-10-25 at 14 14 17](https://user-images.githubusercontent.com/1204802/47499526-75d2a800-d860-11e8-8154-8e43263c959b.png)

Those alignment classes are still not present for images, but I suppose that's a separate issue.